link/uprobe_multi_test.go

```diff
@@ -0,0 +1,159 @@
+package link
+
+import (
+	"errors"
+	"os/exec"
+	"testing"
+
+	"github.com/cilium/ebpf"
+	"github.com/cilium/ebpf/internal/testutils"
+	"github.com/go-quicktest/qt"
+)
+
+func TestUprobeMulti(t *testing.T) {
+	testutils.SkipIfNotSupported(t, haveBPFLinkUprobeMulti())
+
+	prog := mustLoadProgram(t, ebpf.Kprobe, ebpf.AttachTraceUprobeMulti, "")
+
+	um, err := bashEx.UprobeMulti([]string{"bash_logout", "main"}, prog, nil)
```

Comment:
Nit: Can you reuse / add to `bashSym` instead please?

Reply:
ok